src/shared/lib/mail/sendEmail.ts

```diff
@@ -1,33 +1,32 @@
-import { SiteConfig } from "@/shared/config/site-config";
-import { env } from "@/env";
-
-import { resend } from "./resend";
+import nodemailer from "nodemailer";
+import { render } from "@react-email/render";
 
-type ResendSendType = typeof resend.emails.send;
-type ResendParamsType = Parameters<ResendSendType>;
-type ResendParamsTypeWithConditionalFrom = [payload: Omit<ResendParamsType[0], "from"> & { from?: string }, options?: ResendParamsType[1]];
+import { env } from "@/env";
 
-/**
- * sendEmail will send an email using resend.
- * To avoid repeating the same "from" email, you can leave it empty and it will use the default one.
- * Also, in development, it will add "[DEV]" to the subject.
- * @param params[0] : payload
- * @param params[1] : options
- * @returns a promise of the email sent
- */
-export const sendEmail = async (...params: ResendParamsTypeWithConditionalFrom) => {
-  if (env.NODE_ENV === "development") {
-    params[0].subject = `[DEV] ${params[0].subject}`;
-  }
+type EmailPayload = {
+  from?: string;
+  to: string;
+  subject: string;
+  text: string;
+  react?: React.ReactNode;
```

Comment:
```suggestion
  react?: React.ReactElement;
```

to avoid this : 

![image](https://github.com/user-attachments/assets/e77b91c1-e664-4309-92a9-9e6b21fb8c18)